src/main/ruby/core/truffle/boot.rb

```diff
@@ -27,6 +27,8 @@ def self.check_syntax(source_or_file)
     1
   end
 
+  private_class_method :check_syntax
```

Comment:
RubyMine's formatter seems to have rather strange defaults. Fine, let's keep it like that for now.